app.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'sinatra/reloader'
+require_relative 'models'
+
+get '/' do
+  redirect '/memos'
+end
+
+get '/memos' do
+  @title = 'memos'
+  @memos = parse_all_data
+  erb :index
+end
+
+get '/memos/new' do
+  @title = 'new'
+  erb :new
+end
+
+post '/memos' do
+  add_new_memo(params)
+  redirect '/memos'
+end
+
+get '/memos/:id/edit' do
+  @title = 'edit'
+  found_data = find_memo_data(params[:id].to_i)
+  @id = found_data[:id]
+  @name = found_data[:name]
+  @text = found_data[:text]
```

Comment:
```suggestion
  @hoge = found_data
```

erbに渡すときもハッシュのままでOKです！

今は3項目で済んでますが、本格的なプログラムになるとこのやり方だと何十個もインスタンス変数を作ることになってしまいます〜。

Reply:
こちらでもハッシュから値を取り出してしまっていました💦
ハッシュのまま、インスタンス変数@memoでerbに受け渡す形に変更しました。https://github.com/SuzukaHori/sinatra_practice/pull/1/commits/faf90b6f2f01b601c0b8d0b0a518e2aaf6d156e1